util/test/run-in-test-venv.bash

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
```

Comment:
Why create a separate run-in-test-venv instead of just having this in run-in-venv?

Reply:
Because the environment variables only apply to test  -- `CHPL_TEST_VENV_DIR` -- there are other parts like `chpl_home_utils.py --test-venv` as well that talk about the "test venv". It would be fine with me to have `CHPL_VENV_DIR` and allow the same thing for the docs venv, but here I am trying to improve errors and recover functionality that was there, not to rearrange all of the enviro vars etc.

Reply:
Ah, yeah -- ok gotcha

Reply:
We'll have to include this file in the release now, right?